### PR TITLE
Prepare 2.0.0

### PR DESCRIPTION
**Pull request by pmalek-sumo**
###### Description

Prepare 2.0.0

---

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Review by perk-sumo — APPROVED** [review]
🎉

File: deploy/helm/sumologic/README.md [hunk 1]
```diff
@@ -164,10 +164,10 @@ Parameter | Description | Default
...
-`fluent-bit.config.inputs` | Configure Fluent-bit Helm sub-chart inputs. | [fluent-bit.config.inputs in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0-rc.3/deploy/helm/sumologic/values.yaml#L826-L891)
-`fluent-bit.config.outputs` | Configure Fluent-bit Helm sub-chart outputs. | [fluent-bit.config.outputs in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0-rc.3/deploy/helm/sumologic/values.yaml#L894-L902)
-`fluent-bit.config.customParsers` | Configure Fluent-bit Helm sub-chart customParsers. | [fluent-bit.config.customParsers in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0-rc.3/deploy/helm/sumologic/values.yaml#L904-L913)
+`fluent-bit.config.service` | Configure Fluent-bit Helm sub-chart service. | [fluent-bit.config.service in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L816-L824)
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/README.md:167]:
```suggestion
`fluent-bit.config.service` | Configure Fluent-bit Helm sub-chart service. | [fluent-bit.config.service in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L817-L827)
```

File: deploy/helm/sumologic/README.md [hunk 2]
```diff
@@ -164,10 +164,10 @@ Parameter | Description | Default
...
+`fluent-bit.config.inputs` | Configure Fluent-bit Helm sub-chart inputs. | [fluent-bit.config.inputs in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L826-L891)
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/README.md:168]:
```suggestion
`fluent-bit.config.inputs` | Configure Fluent-bit Helm sub-chart inputs. | [fluent-bit.config.inputs in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L828-L895)
```

File: deploy/helm/sumologic/README.md [hunk 3]
```diff
@@ -164,10 +164,10 @@ Parameter | Description | Default
...
+`fluent-bit.config.outputs` | Configure Fluent-bit Helm sub-chart outputs. | [fluent-bit.config.outputs in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L894-L902)
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/README.md:169]:
```suggestion
`fluent-bit.config.outputs` | Configure Fluent-bit Helm sub-chart outputs. | [fluent-bit.config.outputs in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L896-L906)
```

File: deploy/helm/sumologic/README.md [hunk 4]
```diff
@@ -164,10 +164,10 @@ Parameter | Description | Default
...
+`fluent-bit.config.customParsers` | Configure Fluent-bit Helm sub-chart customParsers. | [fluent-bit.config.customParsers in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L904-L913)
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/README.md:170]:
```suggestion
`fluent-bit.config.customParsers` | Configure Fluent-bit Helm sub-chart customParsers. | [fluent-bit.config.customParsers in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L907-L917)
```

**Comment by perk-sumo** [comment]
Good catch @sumo-drosiek  - let's add those docs changes as a first commit and the `rc-3` removal as a second one.

**Comment by perk-sumo** [comment]
That being said - it's so wrong we have to remember about this 🤦

**Comment by justrelax19** [comment]
Looks good.  
The only thing which is remaining is this bug https://jira.kumoroku.com/jira/browse/SUMO-155633.
So, we need to take a final decision on it. cc @perk @pmalek-sumo 

Other than that, I am fine releasing it :greenball:

**Review by kkujawa-sumo — APPROVED** [review]

**Comment by perk-sumo** [comment]
@justrelax19 the issue was fixed in this PR: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1337

**Pull request merged**